README.md

```diff
@@ -1 +1,127 @@
-# Leaderboard-list
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Future Features](#future-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] It would be nice if you fix the Live Demo link in the Table of Contents because right now it is broken 😨 Kindly fix it or remove it if you don't have any section for this link 👍 to make your readme descriptive and professional 😊